src/frontend/initializeNodejs.ts

```diff
@@ -19,5 +21,11 @@ export async function initializeNodejs() {
     }
   }
 
-  nodejs.startWithArgs(`loader.js --rootKey=${rootKey}`);
+  const flags = [`--rootKey=${rootKey}`];
+
+  if (EXTERNAL_FILES_DIR) {
+    flags.push(`--sharedStoragePath=${EXTERNAL_FILES_DIR}`);
+  }
+
+  nodejs.startWithArgs(`loader.js ${flags.join(' ')}`);
```

Comment:
@achou11 I remember that we had this conversation a while ago but don't remember the details. `startWithArgs` is not in the [docs](https://nodejs-mobile.github.io/docs/api/react-native-bridge). Can you remind me what this Api is?

Reply:
it's a way of starting a node js process that will pass provided CLI flags to the process.